airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -776,6 +776,15 @@
           type: "array"
           uniqueItems: true
           order: 7
+        look_back_window:
```

Comment:
you don't need to touch this file, it's done automatically by publishing scripts